dss/events/handlers/index.py

```diff
@@ -45,26 +58,43 @@ def is_bundle_to_index(key) -> bool:
 
 def read_bundle_manifest(s3, bucket_name, bundle_key, logger):
     manifest_string = s3.Object(bucket_name, bundle_key).get()['Body'].read().decode("utf-8")
-    logger.debug("Read bundle manifest from bucket %s with bundle key %s: %s", bucket_name, bundle_key, manifest_string)
+    logger.debug(f"Read bundle manifest from bucket {bucket_name}"
+                 f" with bundle key {bundle_key}: {manifest_string}")
     manifest = json.loads(manifest_string, encoding="utf-8")
     return manifest
 
 
-def create_index_data(s3, bucket_name, manifest, logger):
+def create_index_data(s3, bucket_name, bundle_id, manifest, logger):
     index = dict(state="new", manifest=manifest)
-    files_info = manifest['files']
+    files_info = manifest[BundleMetadata.FILES]
     index_files = {}
     bucket = s3.Bucket(bucket_name)
     for file_info in files_info:
-        if file_info['indexed'] is True:
-            if not file_info["name"].endswith(".json"):  # TODO Don't check filename, if file is 'indexed' load json
-                logger.warning("File %s is marked for indexing but does not end in .json. It will not be indexed.",
-                               file_info["name"])
+        if file_info[BundleFileMetadata.INDEXED] is True:
+            if file_info[BundleFileMetadata.CONTENT_TYPE] != 'application/json':
+                logger.warning((f"In bundle {bundle_id} the file \"{file_info[BundleFileMetadata.NAME]}\""
```

Comment:
The inner parens were needed before conversion to f-strings. They are cleaned-up now.